toasty/simple_interface.py

```diff
@@ -0,0 +1,51 @@
+from toasty import builder, collection, pyramid, multi_tan, multi_wcs
+import reproject
+
+
+def toast_fits_list(fits_list, **kwargs):
+    """
+    Process a file or a list of FITS files into a tile pyramid with
+    a common tangential projection.
+
+    Parameters
+    ----------
+    fits_list : str or list of str
+        A single path or a list of paths to FITS files to be processed.
+    kwargs
+        Settings for the `toasty` tiling process. Common
+        settings include 'hdu_index' and 'blankval'.
+
+    Returns
+    -------
+    out_dir : :class:`str`
+        The relative path to the base directory where the tiled files are located
+    image_set_pattern : :class:`str`
+        The pattern to be used to access specific tiles
+    """
+
+    if not isinstance(fits_list, list):
+        fits_list = [fits_list]

```

Comment:
I think it would be better to test whether the first argument is a string, rather than test whether it's specifically a `list`, since someone might also pass in a tuple, a generator of filenames, etc. Really we want to test whether the first argument is something iterable, but unfortunately that doesn't work here because Python strings are considered iterable.